docs/rules/sort-imports.md

```diff
@@ -136,6 +138,34 @@ import c from 'baz.js';
 
 Default is `false`.
 
+### `ignoreDeclarationSort`
+
+Ignores the sorting of import declaration statements.
```

Comment:
Please add a note that the default is false here. (I see the note below, but I think it's important enough to be included in the intro/summary of the section.)

Reply:
I decided to keep it at the end for consistency with the other sections.

Please let me know if this really needs to be changed.